API/Data/Migrations/20230205222635_EntityTimestampsUtc.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace API.Data.Migrations
+{
+    public partial class EntityTimestampsUtc : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedUtc",
+                table: "Volume",
+                type: "TEXT",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "LastModifiedUtc",
+                table: "Volume",
+                type: "TEXT",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```

Comment:
You will need to setup a converter for the Utc Fields so that when they are read in, the DateTimeKind is set.

See https://stackoverflow.com/questions/4648540/entity-framework-datetime-and-utc/73154546#73154546

Reply:
ok